package.json

```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "@aws-sdk/client-eventbridge": "^3.577.0",
     "@aws-sdk/util-dynamodb": "^3.577.0",
+    "@dvsa/cvs-type-definitions": "^7.2.0",
```

Comment:
This version doesn't exist yet, it won't be until CB2-12693 is merged in.

Reply:
It is 7.2 in develop?

Reply:
Apologies, forgot the change had already been merged in.